cli/src/alluxio.org/cli/cmd/info/report.go

```diff
@@ -80,9 +84,134 @@ func (c *ReportCommand) Run(args []string) error {
 		}
 		reportArg = args[0]
 	}
-	// TODO: output all in a serializable format and filter/trim as specified by flags
-	if c.format != "json" && c.format != "yaml" {
-		return stacktrace.NewError("Invalid format %v, must be one of [json, yaml]", c.format)
+
+	buf := &bytes.Buffer{}
+	if err := c.RunWithIO([]string{reportArg}, nil, buf, os.Stderr); err != nil {
+		io.Copy(os.Stdout, buf)
+		return err
+	}
+
+	obj := orderedmap.New()
+	if err := json.Unmarshal(buf.Bytes(), &obj); err != nil {
+		return stacktrace.Propagate(err, "error unmarshalling json from java command")
+	}
+
+	if reportArg == "summary" {
+		if val, ok := obj.Get("freeCapacity"); ok {
```

Comment:
i suggest using suffixes like "Bytes", "Milliseconds", and "Time" to avoid using a static table of predefined keys. in this way, we can automatically handle the key changes. i think `orderedmap` already handles the nested map, we can recursively convert all the objects in the json.